src/helpers/crypto.cpp

```diff
@@ -1,12 +1,12 @@
 #include "helpers/crypto.h"
 
 #include <cassert>
-#include "bcl/Ecdsa.hpp"
-#include "bcl/Sha256.hpp"
-#include "bcl/Uint256.hpp"
+#include "lib/bcl/Ecdsa.hpp"
```

Comment:
The pattern of #include "lib/*" is not really done.  The lib directory should be added as an include path in the project files.

e.g. -Isrc/lib

#include "bcl/*.hpp"

Reply:
Makes sense, thanks for the good catch 👍 

Got it building with `bcl/*.hpp`, I just need to update the Arduino script next.

Reply:
Sounds good.  Thanks.